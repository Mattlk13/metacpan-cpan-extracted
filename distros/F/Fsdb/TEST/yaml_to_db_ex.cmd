prog='yaml_to_db'
args=''
cmp='diff -c -b '
requires='YAML::XS'
