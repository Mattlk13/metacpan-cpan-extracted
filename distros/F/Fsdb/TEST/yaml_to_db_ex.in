- name: ACM
  role: sponsor
  alttext: ACM, the Association for Computing Machinery
  image: logos/acm-small.jpg
  link:  https://www.acm.org/
  date: 2016-01-01

- name: SIGCOMM
  role: sponsor
  alttext: SIGCOMM, ACM'S Special Interest Group on Communication
  image: logos/sigcommlogo.png
  link:  http://sigcomm.org
  date: 2016-01-02

- name: SIGMETRICS
  role: sponsor
  alttext: SIGMETRICS, ACM'S Special Interest Group on Performance Evaluation
  image: logos/sigmetrics-small.png
  link:  http://www.sigmetrics.org
  date: 2016-01-03
