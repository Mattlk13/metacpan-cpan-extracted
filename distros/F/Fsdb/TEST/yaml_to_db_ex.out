#fsdb -F t alttext date image link name role
ACM, the Association for Computing Machinery	2016-01-01	logos/acm-small.jpg	https://www.acm.org/	ACM	sponsor
SIGCOMM, ACM'S Special Interest Group on Communication	2016-01-02	logos/sigcommlogo.png	http://sigcomm.org	SIGCOMM	sponsor
SIGMETRICS, ACM'S Special Interest Group on Performance Evaluation	2016-01-03	logos/sigmetrics-small.png	http://www.sigmetrics.org	SIGMETRICS	sponsor
#   | yaml_to_db
